### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by hardillb**
## Description

Add more tests

### CODE REVIEW

File: test/unit/configmap_spec.js [hunk 1]
```diff
@@ -85,14 +90,14 @@ describe('Examine Config Maps',  function () {
...
-                ingress.should.have.length(1)
-            })
+            // it('has service', function () {
+            //     const service = services.filter( s => s.metadata.name === 'flowforge-broker')
+            //     service.should.have.length(1)
+            // })
+            // it('has ingress', function () {
+            //     const ingress = ingresses.filter( s => s.metadata.name === 'flowforge-broker')
+            //     ingress.should.have.length(1)
+            // })
```
Comment (Steve-Mcl) [inline test/unit/configmap_spec.js:100]:
```suggestion
```

Looks like these have been moved to individual _spec files - remove commented code

File: test/unit/configmap_spec.js [hunk 2]
```diff
@@ -6,14 +6,13 @@ describe('Examine Config Maps',  function () {
...
     before(async function () {
        const { docs } = await setup()
-       configMaps = docs.filter(d => d.kind === 'ConfigMap')
-       services = docs.filter(d => d.kind === 'Service')
-       ingresses = docs.filter(d => d.kind === 'Ingress')
-       deployments = docs.filter(d => d.kind === 'Deployment')
+       configMaps = docs.filter(d => d && d.kind === 'ConfigMap')
+       services = docs.filter(d => d && d.kind === 'Service')
+       ingresses = docs.filter(d => d && d.kind === 'Ingress')
+       deployments = docs.filter(d => d && d.kind === 'Deployment')
```
Comment (Steve-Mcl) [inline test/unit/configmap_spec.js:15]: Looks like `let ingresse` declaration has been deleted?

File: test/unit/configmap_spec.js [hunk 3]
```diff
@@ -6,14 +6,13 @@ describe('Examine Config Maps',  function () {
...
     let deployments
     before(async function () {
        const { docs } = await setup()
-       configMaps = docs.filter(d => d.kind === 'ConfigMap')
-       services = docs.filter(d => d.kind === 'Service')
-       ingresses = docs.filter(d => d.kind === 'Ingress')
-       deployments = docs.filter(d => d.kind === 'Deployment')
+       configMaps = docs.filter(d => d && d.kind === 'ConfigMap')
+       services = docs.filter(d => d && d.kind === 'Service')
+       ingresses = docs.filter(d => d && d.kind === 'Ingress')
```
Comment (hardillb) [inline test/unit/configmap_spec.js:14]:
```suggestion
```

Comment (Steve-Mcl) [inline test/unit/configmap_spec.js]:
```suggestion
```

`services` and `ingresse` appear to be unused in `test/unit/helm_spec.js`

If kept for future use, then it looks like the decleration `let ingresse` has been deleted.

**Review by Steve-Mcl — COMMENTED** [review]
Nothing much - mostly tidy up.

**Review by Steve-Mcl — APPROVED** [review]

**Pull request merged**